worker/machiner/machiner.go

```diff
@@ -139,11 +139,12 @@ func setMachineAddresses(tag names.MachineTag, m Machine) error {
 	}
 	// Filter out any LXC or LXD bridge addresses.
 	hostAddresses = network.FilterBridgeAddresses(hostAddresses)
-	logger.Infof("setting addresses for %v to %q", tag, hostAddresses)
+	logger.Infof("setting addresses for %q to %v", tag, hostAddresses)
 	return m.SetMachineAddresses(hostAddresses)
 }
 
 func (mr *Machiner) Handle(_ <-chan struct{}) error {
+	logger.Infof("got a Machiner.Handle() event")
```

Comment:
nope, it was a debug at one point in the past while I figured out what was going on, should be just removed.